### DISCUSSION TITLE
Satellite maps for base map?

### DISCUSSION THREAD

**Discussion by pyssling** [OpenOrienteering/mapper#2192] [General]
Would anyone be able to share tips on where I can get satellite maps for basemaps, ideally high-resolution and georeferenced? There seems to be a tool called SASPlanet, but I can't find reliable/good instructions or even a reliable download location for it.

**Comment by yevhenmazur** [comment]
[Download location](http://www.sasgis.org/download/)
[Instructions (Ukr lang)](https://yevhen.mazur.blog/2016/10/23/avtomatychna-heopryvyazka-materialiv-z-sas-planet-u-openorienteering-mapper/) - [English google-translate](https://yevhen-mazur-blog.translate.goog/2016/10/23/avtomatychna-heopryvyazka-materialiv-z-sas-planet-u-openorienteering-mapper/?_x_tr_sl=uk&_x_tr_tl=en&_x_tr_hl=uk&_x_tr_pto=wapp)

**Reply by pyssling** [reply]
I get a "Unable to load maps error" when running, any tips?

**Reply by yevhenmazur** [reply]
Please get in touch with me at yevhen.mazur@pm.me. This is not a good place to troubleshoot the SAS planet.

**Reply by pyssling** [reply]
I figured this out, and got it working, something to do with the maps folder. There is a patch circulating on the internet.

**Comment by ollesmaps** [comment]
Some info here:
https://omapwiki.orienteering.sport/tutorials/how-to-make-a-sprint-orienteering-map-beginners-guide/

**Reply by pyssling** [reply]
Thank you! Excellent reference material, exactly what I needed.
